### PR TITLE
Installation procedure for the litmus operator

### PR DESCRIPTION
**Pull request by aabughosh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,6 +53,10 @@ delete-community-operator:
 install-crds:
 	./scripts/deploy-test-crds.sh
 
+install-litmus:
+	kubectl apply -f https://litmuschaos.github.io/litmus/2.7.0/litmus-2.7.0.yaml
```
Comment (hamadise) [inline Makefile:57]: we are using oc not kubectl

File: Makefile [hunk 2]
```diff
@@ -52,6 +53,10 @@ delete-community-operator:
 install-crds:
 	./scripts/deploy-test-crds.sh
 
+install-litmus:
+	oc apply -f https://litmuschaos.github.io/litmus/2.7.0/litmus-2.7.0.yaml
+	oc apply -f ./scripts/operator-ce.yaml
```
Comment (greyerof) [inline Makefile:58]: Move these steps to a ./scripts/deploy-litmus-operator.sh script and call it here, as the rest of this file is doing.

File: Makefile [hunk 3]
```diff
@@ -60,6 +65,10 @@ delete-deployment:
 create-statefulset:
 	./scripts/deploy-statefulset-test-pods.sh
 
+delete-litmus:
+    oc delete -f https://litmuschaos.github.io/litmus/2.7.0/litmus-2.7.0.yaml
+	oc delete -f ./scripts/operator-ce.yaml
```
Comment (greyerof) [inline Makefile:70]: Same here: create a scripts/delete-litmus-operator.sh

File: scripts/operator-ce.yaml [hunk 4]
```diff
@@ -0,0 +1,725 @@
+apiVersion: v1
```
Comment (greyerof) [inline scripts/operator-ce.yaml:1]: A couple of questions regarding this yaml:
1. What does the "-ce" suffix in the file name mean?
2. As we're deploying OLM in our cluster, isn't it possible to deploy this operator using OLM, creating the proper subscription? Is it this one https://artifacthub.io/packages/olm/community-operators/litmuschaos ? The yaml for the OLM subscruption (alpha channel) can be found here: https://operatorhub.io/install/alpha/litmuschaos.yaml

Comment (aabughosh) [reply scripts/operator-ce.yaml:1]: ce is the meaning of chaos engine 
and i tried with the olm it didnt work there is some resources that are missing

File: Makefile [hunk 5]
```diff
@@ -60,6 +64,10 @@ delete-deployment:
 create-statefulset:
 	./scripts/deploy-statefulset-test-pods.sh
 
+delete-litmus:
+	./scripts/delete-litmus-operator.sh
+
+
 # deletes the namespace completely
 clean-all:
```
Comment (hamadise) [inline Makefile:72]: clean-all script should call delete-litmus as well.

**Review by greyerof — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**